### PR TITLE
Fix errors reported by covscan

### PR DESCRIPTION
**Pull request by ifranzki**
Closes: https://github.com/opencryptoki/opencryptoki/issues/782

### CODE REVIEW

File: usr/lib/common/loadsave.c [hunk 1]
```diff
@@ -2848,6 +2848,14 @@ CK_RV load_public_token_objects(STDLL_TokData_t *tokdata)
             continue;
         }
 
+        /* size can not be negative if treated as signed int */
```
Comment (jschmidb) [inline usr/lib/common/loadsave.c:2851]: as unsigned int

Comment (ifranzki) [reply usr/lib/common/loadsave.c:2851]: no, the problem arises if it is treated as **signed** int, because then its value would be negative.

**Review by jschmidb — APPROVED** [review]
ok, lgtm as discussed.

**Comment by ifranzki** [comment]
Close + reopen to trigger Travis

**Pull request merged**